index.html

```diff
@@ -0,0 +1,173 @@
+<!DOCTYPE html>
+<html lang="en" id="html">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>BSC Summer 2021</title>
+    <link rel="icon" href="images/logo.jpg" />
+    <link rel="stylesheet" href="style/style.css" />
+    <link
+      rel="stylesheet"
+      href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.14.0/css/all.min.css"
+    />
+    <link rel="preconnect" href="https://fonts.googleapis.com" />
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Lato:wght@100&display=swap"
+      rel="stylesheet"
+    />
+  </head>
+  <body id="body">
+    <header>
+      <div class="mobile-nav">
+        <i class="fas fa-bars fa-2x" id="hamburger"></i>
+      </div>
+      <nav class="nav-top">
+        <ul class="nav-top-links">
+          <li>
+            <a class="nav-top-links-style" href="#"
+              ><i class="fab fa-facebook-f"></i
+            ></a>
+          </li>
+          <li>
+            <a class="nav-top-links-style" href="#"
+              ><i class="fab fa-twitter"></i
+            ></a>
+          </li>
+          <li><a class="nav-top-links-style" href="#">Bulgarian</a></li>
+          <li><a class="nav-top-links-style" href="#">My page</a></li>
+          <li><a class="nav-top-links-style" href="#">Logout</a></li>
+        </ul>
+      </nav>
+      <nav class="nav-bottom">
+        <div class="nav-bottom-width">
+          <a href="index.html"
+            ><img class="logo" src="images/Salsa.png" alt="Logo"
+          /></a>
+          <ul class="nav-bottom-links">
+            <li class="header-botoom-list">
+              <a class="nav-bottom-links-style" href="about.html">About</a>
+            </li>
+            <li class="header-botoom-list">
+              <a class="nav-bottom-links-style" href="#program">Program</a>
+            </li>
+            <li class="header-botoom-list">
+              <a class="nav-bottom-links-style" href="#sponsors">Sponsor</a>
+            </li>
+            <li class="header-botoom-list">
+              <a class="nav-bottom-links-style" href="#featureDancer">News</a>
+            </li>
+            <li class="header-botoom-list">
+              <a class="nav-bottom-links-campaing" href="#">Join</a>
+            </li>
+          </ul>
+        </div>
+      </nav>
+    </header>
+    <section class="section-header">
```

Comment:
- [ ] **The font hierarchy and background in your hero section don't match the original. Please modify it.**

Original Design|Cloned Desing
:-------------------:|:------------------:
![image](https://user-images.githubusercontent.com/59168430/124023509-a4d74980-da0b-11eb-8efe-aeb79ec9464f.png)|![image](https://user-images.githubusercontent.com/59168430/124023624-c0daeb00-da0b-11eb-8a3f-9de810679a89.png)

Following a design guideline is one of the most important skills for a software developer. You will always encounter it in your future jobs. So, let us achieve mastery at it. Please make sure that the font hierarchy(weight, size, etc) matches the original design. Also, the background on cloned design is different from the original.

Reply:
Hello Anas,

as we have to personalize the project, I changed the topic of the webpage. This background image should be personalized as well. Do I really need to use the SAME background like in the original design?

Reply:
@PolinaStamenova  

Hey, 

I checked the requirements. There is no hard requirement regarding the background image. So, I guess you can keep it the way you want. But, please make sure that your design matches the following. 

    Colors.
    Typography: font face, size, and weight.
    Layout: composition and space between elements.

Also, personalizing content means that the fundamental idea of your website is personalized. The design should still be followed. Nonetheless, as there is no hard requirement for the background image, you can keep it. I am sorry for requesting it in the first place. 

Please feel free to leave any questions or comments. Good Luck!

Reply:
Hey Anas,

thank you for the fast replay! 😄

Reply:
Solved issue